.github/workflows/pr-test-xpu.yml

```diff
@@ -0,0 +1,156 @@
+# TODO: this looks sort of similar to _linux-test, but there are like a dozen
+# places where you would have to insert an if statement. Probably it's better to
+# just use a different workflow altogether
+
+name: xpu-test
+
+on:
+  push:
+    branches:
+      - main
+      - 'gh/**'
+  pull_request:
+    branches:
+      - main
+      - 'gh/**'
```

Comment:
```suggestion
      - ciflow/xpu/*
```

Reply:
Please add `ciflow/xpu` in https://github.com/pytorch/ao/blob/main/.github/pytorch-probot.yml